app/registries/foreman/plugin.rb

```diff
@@ -49,6 +51,8 @@ class << self
       attr_accessor :tests_to_skip, :report_scanner_registry,
         :report_origin_registry, :medium_providers,
         :graphql_types_registry
+      attr_accessor :fact_registry
```

Comment:
Any particular reason not to list the `:fact_registry` with the other `attr_accessor`s?